tests/reg_tests/test_DVGeometryCST.py

```diff
@@ -23,13 +24,26 @@
 except ImportError:
     prefoilImported = False
 
+baseDir = os.path.dirname(os.path.abspath(__file__))
+inputDir = os.path.join(baseDir, "../../input_files")
```

Comment:
If I were being pedantic, I'd say that this should be `os.path.join(baseDir, "..", "..", "input_files")` instead because I believe the forward slashes won't work on all operating systems.